<?xml version="1.0"?>
<launch>

    <param name="maxlen"               type="double" value="100.0" />    
    <param name="minlen"               type="double" value="0.001" />    
    <param name="angular_resolution_x" type="double" value="0.5" />
     <param name="angular_resolution_y" type="double" value="1.025"/> 
    <param name="max_angle_width"      type="double" value="360" />
    <param name="max_angle_height"     type="double" value="180" />
    <param name="pcTopic"       type="str"    value="/velodyne_points" />

    <node pkg="pc2image" type="zdepthdata_node" name="pc2image_node" />
    <node pkg="pc2image" name="filter_node" type="filter_PCL.py" />

    <arg name="rviz" default="false" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find pc2image)/rviz/pc2image_rviz.rviz" />
    </group>
  
</launch>

